src/components/currencies/currencyListing.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../common/colors.scss';
+
+#card {
+  // background-color: rgba(41, 76, 180, 0.8);
+}
```

Comment:
- [ ] Kindly remove this file if it is not being used. This will make your codebase cleaner and more professional